scripts/setup.sh

```diff
@@ -186,6 +186,9 @@ if [ ! -d "$NODE_PATH" ]; then
   chmod -R 744 $NODE_MODULE_DIR
 fi
 
+# Change execute status of browsermob-proxy
+chmod +x $TOOLS_DIR/browsermob-proxy-2.1.1/bin/browsermob-proxy
```

Comment:
We don't have much control over the zip file, though. I also did notice that a lot of our scripts do already have the execution bit flipped, so maybe this isn't too bad. Let's keep this, for now. I want to discuss it with the other TLs.